pkg/rhelv2/rpm/rpm.go

```diff
@@ -77,33 +84,31 @@ func AddToDependencyMap(filename string, fileData analyzer.FileData, execToDeps,
 	}
 }
 
-func getCPEsUsingEmbeddedContentSets(files analyzer.Files) ([]string, error) {
-	defer metrics.ObserveListFeaturesTime(pkgFmt, "cpes", time.Now())
-
-	// Get CPEs using embedded content-set files.
-	// The files are stored in /root/buildinfo/content_manifests/ and will need to
-	// be translated using mapping file provided by Red Hat's PST team.
-	contents := getContentManifestFileContents(files)
+// getContentManifestSets returns the list of content sets defined in the content
+// manifest, if found in the files. Otherwise, `nil` or empty slice is returned.
+func getContentManifestSets(files analyzer.Files) ([]string, error) {
+	var contents []byte
+	for _, prefix := range contentManifestDirs.AsSlice() {
+		for name, file := range files.GetFilesPrefix(prefix) {
+			if strings.HasSuffix(name, ".json") {
+				// Return the first one found, as we are currently assuming there is only one per
```

Comment:
This is what I'm wondering about. Is this a valid assumption for RHCOS nodes? Not sure if this is something confirmed by the RHCOS team

Reply:
Ignore this, I didn't see you followed up in the original thread. I'll continue the conversation there https://github.com/stackrox/scanner/pull/1055#discussion_r1072957726